data/githubColors.json

```diff
@@ -0,0 +1,2142 @@
+{
```

Comment:
Where did you get this file?

* If you got this file from someone, we should definitely credit them
* Could we somehow make this a package or dependency instead? While I'm wary of NPM, I feel like this kind of thing shouldn't be committed to our repository/rely on us to update.

Reply:
The file is from someone's [github-colors repository](https://github.com/ozh/github-colors) but it's not an npm package. While it doesn't have an NPM package, it is automatically updated. Instead of putting in the json file ourselves, we can also run a get request on the raw json file from the repository at build time instead with the [raw content](https://raw.githubusercontent.com/ozh/github-colors/master/colors.json) link.

Reply:
I'm going to grab from the raw content link but I'll use the .json as a fallback.

Reply:
Agh sorry, I meant to respond to this in my earlier review run - I think we should:

* definitely credit them somewhere, probably in the code and the README
* consider a more sustainable way to do this than the raw content link or including it in our repo; a submodule comes to mind, though I'm not sure how that would factor in to our build step.